andela/denethor-ah-backend
# 163478813 implement user can report articles

#### What does this PR do?
* Implement user can report an article

#### Description of Task to be completed?
* write tests for user can report an article
* Implement user can report an article
* implement user can report an article only once

#### How should this be manually tested?
* clone the repo and pull this branch `feature/163478813/user-can-report-articles`
* create a user using the endpoint `signup`POST `/api/users`
* verify the user using the verify link returned on signup and retrieve the token returned on successful user verification.
* create an article and generate an articleId POST `/api/articles`
* use the token of a verified user to report an article by hitting the `/api/articles/:id/report`
* Report article by different users to ensure the article is deleted at the 5th report.

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163478813
